tests/helperDb.js

```diff
@@ -14,7 +13,7 @@ var courseDB = require('../lib/course-db');
 var postgresqlUser = 'postgres';
 var postgresqlDatabase = 'pltest';
 var postgresqlHost = 'localhost';
-var initConString = 'postgres://localhost/postgres';
+var initConString = 'postgres://postgres@localhost/postgres';
```

Comment:
This must've been left over from when I was having issues with Postgres, it works fine when I remove. I'll get rid of it!